cern_search_rest_api/modules/cernsearch/jsonschemas/indico/indico/attachments_v1.0.0.json

```diff
@@ -33,44 +33,61 @@
         }
       }
     },
-    "id": {
-      "type": "string",
-      "description": "Attachments id."
-    },
-    "category_path": {
-      "type": "array",
-      "items": {
-        "type": "string"
-      },
-      "description": "Attachments category path (ordered array)."
-    },
-    "event_id": {
-      "type": "string",
-      "description": "Event id."
-    },
-    "contribution_id": {
-      "type": "string",
-      "description": "Contribution id.",
-      "required": false
-    },
-    "subcontribution_id": {
-      "type": "string",
-      "description": "Event id.",
-      "required": false
-    },
+    "_data": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Attachments id."
+        },
+        "category_path": {
+          "type": "array",
+          "items": {
+            "type": "string"
+            },
+          "description": "Attachments category path (ordered array)."
+        },
+        "event_id": {
+          "type": "string",
+          "description": "Event id."
+        },
+        "contribution_id": {
+          "type": "string",
+          "description": "Contribution id.",
+          "required": false
+        },
+        "subcontribution_id": {
+          "type": "string",
+          "description": "Event id.",
+          "required": false
+        },
+        "filename": {
+          "type": "string",
+          "description": "Attachments filename."
+        },
+        "content": {
+          "type": "string",
+          "description": "Attachments content."
+        },
+        "url": {
+          "type": "string",
+          "description": "Full path Attachment URL"
+        }
+      }
+	},
     "creation_date": {
       "type": "string",
       "description": "Attachments creation date."
     },
-    "filename": {
+    "_updated": {
```

Comment:
Please remove the definitions for `_updated`, `_created` and `control_number` in the **`jsonschemas` mappings only** as they are self generated there's no need to run validations o them.